packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -37,6 +37,9 @@ export class ChoiceList extends React.PureComponent {
       choiceProps.onChange = this.props.onChange;
       choiceProps.size = this.props.size;
       choiceProps.type = this.props.type;
+      choiceProps.inputClassName = classNames(choiceProps.inputClassName, {
+        'ds-c-choice--error': this.props.errorMessage,
```

Comment:
I think we probably have to use `classNames` here too, in case the user specifies an `inputClassName` for the Choice as well. I think this is just one of the problems with not having a separate prop for error :/